### PR TITLE
Feat: registerFaustBlock helper

### PR DESCRIPTION
**Pull request by theodesp**
## Tasks

- [x ] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.
- [ ] If a code change, I have written testing instructions that the whole team & outside contributors can understand.
- [ ] I have written and included a comprehensive [changeset](https://github.com/wpengine/faustjs/blob/canary/DEVELOPMENT.md#deployment) to properly document the changes I've made.

## Description

* Includes a new helper function `registerFaustBlock` that wraps `registerBlockType` function.
* Adds unit tests.

### CODE REVIEW

File: examples/next/getting-started/tsconfig.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "baseUrl": "./src",
-    "incremental": true
+    "incremental": true,
+	"typeRoots": ["node_modules/@types"]
```
Comment (theodesp) [inline examples/next/getting-started/tsconfig.json:22]: Prevents type conflicts between different React Versions. So we need to specify explicitly the location of `typeRoots`.

File: packages/block-editor-utils/src/types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare global {
+  let wp: unknown;
```
Comment (theodesp) [inline packages/block-editor-utils/src/types/index.d.ts:2]: Global `wp` version is available only within the WordPress installation context.

**Review by blakewilson — APPROVED** [review]
Nice @theodesp! Looks good. We may not need the changesets for `@faustwp/core` and `@faustjs/next` since it's just dev related, but I'll let you make the final call.

File: packages/faustwp-cli/src/telemetry/marshallTelemetryData.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import fs from 'fs';
-import { platform } from 'node:process';
+import { platform } from 'process';
```
[the remark this answers is not in the document]
Comment (blakewilson) [reply packages/faustwp-cli/src/telemetry/marshallTelemetryData.ts:2]: Just adding this link here for context, `node:process` and `process` resolve to the same thing:

https://stackoverflow.com/a/71473519

**Comment by theodesp** [comment]
> Nice @theodesp! Looks good. We may not need the changesets for `@faustwp/core` and `@faustjs/next` since it's just dev related, but I'll let you make the final call.

I will add them just in case we need to do a release. It's a very small change anyway.

**Pull request merged**